### ISSUE TITLE
Filters are not recognized

### ISSUE THREAD

**Issue by utoc** [MattCzyr/ConsoleFilter#1]
Hi there!

I'm having some issues with your mod that I would like to report.  My current config:

(the (#) annotations are NOT in the actual config)

```
general {
    # Any console messages containing one of these strings will be hidden.
    S:messagesToFilter <
    Player moved too quickly
    Sending server configs to client
    Sending Player Data to player
    Sent config to 
    Client attempting to join with (4)
    mcjty.lib.preferences.PreferencesProperties (2)
    techreborn.multiblocks.MultiBlockCasing (1)
    Rcon connection from (3)
    ElevatorButtonScreenModule
    is sleeping.
    Unloading WorldData
    joining dimension
     >
}



```

Lines that aren't working:

```
(1)
[Server thread/INFO] [STDOUT]: [techreborn.multiblocks.MultiBlockCasing:isBlockGoodForInterior:303]: Block{minecraft:air}

(2)
[Server thread/INFO] [STDOUT]: [mcjty.lib.preferences.PreferencesProperties:syncToClient:31]: syncToClient: style = STYLE_FLAT_GRADIENT

(3)
[RCON Listener #1/INFO]: Rcon connection from: /127.0.0.1

(4)
Client attempting to join with 100 mods : 
```

I can't find a pattern or reason as why some filters work and others dont.  There are others that match and don't match.. I followed your example in creating my config without quotations or separating each line with a comma.  Thoughts?

**Comment by MattCzyr** [comment]
Currently, Console Filter has two independent filters working simultaneously to find messages that should be hidden: one for [Java's Logger](https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html) and one for [Apache Log4j's Logger](https://logging.apache.org/log4j/2.0/log4j-api/apidocs/org/apache/logging/log4j/Logger.html). These are typically the only two loggers that get used in modding.

The first and second messages come from direct calls to System.out.println, not from a logger. Whenever you see a fully qualified class name before a message, that's quite likely coming from a direct call like this. This is not so easily filtered, but it can be done.

The fourth message comes from Forge's custom logger, which appears to be neither of the aforementioned loggers. It's likely the same deal for the third message. I don't think there's really anything to be done here.

I'll see if I can get a filter for System.out.println calls working in the next update.

**Comment by DarianLStephens** [comment]
I'm guessing there hasn't been any luck with that?
This would be very useful for abandoned/old version of mods which left debug lines in; sometimes, the console will be spammed so badly that I get several megabytes worth of log in one play session.

**Comment by MattCzyr** [comment]
Fixed as of version 1.12.2-1.1.0.

**Comment by DarianLStephens** [comment]
I guess, if this is closed, there's no hope for this fix coming to older versions of Minecraft?

**Comment by MattCzyr** [comment]
How old are we talking? At this point I generally don't support older versions for any of my mods, but I could make an exception if there's a specific version you're looking for.

**Comment by DarianLStephens** [comment]
I was hoping for 1.7.10, since there are quite a few mods that have uncontrollable debug outputs that really hurt their usability. (Hundreds of lines every few seconds can add up quick, and the logs are nigh-unreadable)
Unfortunately, most seem to use the system.out thing, which makes this totally helpless.

**Comment by MattCzyr** [comment]
I've released version 1.1.1 (which includes the System.out fix) for 1.7.10. Enjoy!
